Packs/VersaDirector/Integrations/VersaDirector/VersaDirector.py

```diff
@@ -0,0 +1,3810 @@
+        readable_output="Command run successfully.",
+    )
+    return command_results
+
+
+def template_access_policy_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    organization_args = args.get("organization")
+    template_name = args.get("template_name", "")
+    page = arg_to_number(args.get("page"))
+    page_size = arg_to_number(args.get("page_size"))
+    limit = arg_to_number(args.get("limit"))
+    offset = set_offset(page, page_size)
+
+    check_limit(limit)
+
+    organization = set_organization(organization_args, client.organization_params)
+
+    response = client.template_access_policy_list_request(organization, template_name, offset, limit)
+    response = json.loads(xml2json((response.content)))
+
+    command_results = CommandResults(
+        outputs_prefix=VENDOR_NAME + ".TemplateAccessPolicy",
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown(
+            name=f"Access policies associated with {organization}",
+            t=response.get("collection", {}).get("access-policy-group", {}),
+            headerTransform=pascalToSpace,
+            headers=["name", "rules"],
+            is_auto_json_transform=True,
+        ),
+    )
+    return command_results
+
+
+def template_access_policy_rule_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    organization_args = args.get("organization")
+    template_name = args.get("template_name", "")
+    access_policy_name = args.get("access_policy_name", "")
+    page = arg_to_number(args.get("page"))
+    page_size = arg_to_number(args.get("page_size"))
+    limit = arg_to_number(args.get("limit"))
+    offset = set_offset(page, page_size)
+
+    check_limit(limit)
+
+    organization = set_organization(organization_args, client.organization_params)
+
+    response = client.template_access_policy_rule_list_request(organization, template_name, access_policy_name, offset, limit)
+    response = json.loads(xml2json((response.content)))
+
+    command_results = CommandResults(
+        outputs_prefix=VENDOR_NAME + ".TemplateAccessPolicyRule",
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown(
+            name=f"Access policies associated with {organization}",
+            t=response.get("collection", {}).get("access-policy", {}),
+            headerTransform=pascalToSpace,
+            headers=["name", "description", "tag", "rule-disable"],
+            is_auto_json_transform=True,
+        ),
+    )
+    return command_results
+
+
+def template_access_policy_rule_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    organization_args = args.get("organization", "")
+    template_name = args.get("template_name", "")
+    access_policy_name = args.get("access_policy_name", "")
+
+    _args = get_access_policy_rule_args_with_possible_custom_rule_json(args)
```

Comment:
Please fix this as we talked - change naming and use **.
(On all other places using this function and `get_sdwan_policy_rule_args_with_possible_custom_rule_json`.)

Reply:
done